Sources/Common/DataModel/BoundingBox/index.js

```diff
@@ -148,6 +148,28 @@ export function getCenter(bounds) {
   ];
 }
 
+export function scaleAboutCenter(bounds, sx, sy, sz) {
+  if (!isValid(bounds)) {
+    return false;
+  }
+  const center = getCenter(bounds);
+  bounds[0] -= center[0];
+  bounds[1] -= center[0];
+  bounds[2] -= center[1];
+  bounds[3] -= center[1];
+  bounds[4] -= center[2];
+  bounds[5] -= center[2];
+  scale(bounds, sx, sy, sz);
+  bounds[0] += center[0];
+  bounds[1] += center[0];
+  bounds[2] += center[1];
+  bounds[3] += center[1];
+  bounds[4] += center[2];
+  bounds[5] += center[2];
+
+  return true;
```

Comment:
I think it make more sense to return the bounds rather than a boolean

Reply:
And it looks like you mostly followed Sebastien's earlier implementation when creating the function

Reply:
Regardless, if we change one we should probably change both.

Reply:
Because it's already merged, I'll make a pass on it eventually.

Reply:
Likely an API breaking change so best to do it in a sep MR anyhow, but there was a bug in this change so another MR